### DISCUSSION TITLE
Wrapped line after link results in corrupted link

### DISCUSSION THREAD

**Discussion by JohnStephenYeo** [sumatrapdfreader/sumatrapdf#3316] [General]
I have a pdf I'm viewing. It has a line that wraps after the url. When I hover over the url or click on the url, it seems to pull the first word from the following line into the url so that the link doesn't work.

It isn't a pdf that I created so I can't see what is happening where the line wrap. This does not occur in the Adobe reader or in the Chrome reader. The link works fine there.

**Comment by GitHubRulesOK** [comment]
Hmm there are no line wraps in PDF so perhaps that word is trapped in the same object strings but without the sample no way to tell!

**Reply by JohnStephenYeo** [reply]
Would having the pdf be useful? I can send that

**Reply by JohnStephenYeo** [reply]
It might have come from a Word document originally

**Reply by GitHubRulesOK** [reply]
if its restricted/private it needs to go to developer if its small you can post here if its big a clean link to a copy would do.

**Reply by JohnStephenYeo** [reply]
Large and restricted, unfortunately. If you let me know how to send it to the developer, happy to try.

**Reply by GitHubRulesOK** [reply]
kkowalczyk@gmail.com

**Comment by JohnStephenYeo** [comment]
Got it, thanks. 

I very much appreciate your assistance.

Regards
Stephen Yeo
